File: Multiples of 3 or 5 - 6kyu/Python/Multiplesof3or5.py
class Multiplesof3or5(object):
    def solution(self, number):
        if (number < 0): return 0
        factorList = []
        for i in range(1, number):
            if i % 3 == 0:
                factorList.append(i)
            elif i % 5 == 0:
                factorList.append(i)
            else:
                continue
        return sum(i for i in factorList)

File: Multiples of 3 or 5 - 6kyu/Javascript/Multiplesof3or5.js
function solution(number){
  if (number < 0) return 0;
  var sum=0;
  for (var step = 1; step < number; step++) {
    if (step % 3 === 0){
      sum+=step;
      continue;
    }
    else if(step % 5 ===0 ){
      sum+=step;
      continue;
    }
    else
      continue;
  }
  return sum;
}